Explorer/Assets/DCL/SDKComponents/AvatarAttach/Systems/AvatarAttachHandlerSystem.cs

```diff
@@ -0,0 +1,134 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.Throttling;
+using DCL.AvatarRendering.AvatarShape.UnityInterface;
+using DCL.Character.Components;
+using DCL.Diagnostics;
+using DCL.ECSComponents;
+using DCL.SDKComponents.AvatarAttach.Components;
+using DCL.Utilities;
+using ECS.Abstract;
+using ECS.Groups;
+using ECS.LifeCycle;
+using ECS.LifeCycle.Components;
+using ECS.Unity.Transforms.Components;
+using UnityEngine;
+
+namespace DCL.SDKComponents.AvatarAttach.Systems
+{
+    [UpdateInGroup(typeof(SyncedPostRenderingSystemGroup))]
+    [LogCategory(ReportCategory.AVATAR_ATTACH)]
+    [ThrottlingEnabled]
+    public partial class AvatarAttachHandlerSystem : BaseUnityLoopSystem, IFinalizeWorldSystem
+    {
+        private AvatarBase ownPlayerAvatarBase;
+
+        public AvatarAttachHandlerSystem(World world, WorldProxy globalWorld) : base(world)
+        {
+            globalWorld.Query(new QueryDescription().WithAll<PlayerComponent, AvatarBase>(), (ref PlayerComponent playerComponent, ref AvatarBase avatarBase) => { ownPlayerAvatarBase = avatarBase; });
```

Comment:
You can rely that the player entity is never changed and persistent: thus, you don't need to query it in every scene: e.g. you can place `playerEntity` directly to `WorldProxy`

Reply:
Do you mean something like this? 

https://github.com/decentraland/unity-explorer/pull/267/commits/707fc02479fbbd0bc27babb8a5b5c1d4477ef38b

Otherwise I don't understand what you mean...

Due to [this](https://github.com/decentraland/unity-explorer/pull/267#discussion_r1468337649) I can't initialize the main player AvatarBase in a different point of all the systems/plugins initialization